Library/Styles/ProjectStyles.swift

```diff
@@ -3,9 +3,11 @@ import Prelude
 import Prelude_UIKit
 import UIKit
 
-public func projectAttributedNameAndBlurb(project: Project) -> NSAttributedString {
+public func projectAttributedNameAndBlurb(_ project: Project) -> NSAttributedString {
   let isProjectNamePunctuated = project.name.utf16.last
-    .map { NSCharacterSet.punctuationCharacterSet().characterIsMember($0) } ?? false
+    .flatMap(UnicodeScalar.init)
+    .map(CharacterSet.punctuationCharacters.contains)
+    .coalesceWith(false)
```

Comment:
had to learn a lil something about swift 3 strings for this. could use some 👀 to spot check it